pumps/graph_mongo.go

```diff
@@ -0,0 +1,145 @@
+package pumps
```

Comment:
How do we test the business logic in `graph_mongo.go`?

Reply:
Updated the tests